proc/proc_windows.go

```diff
@@ -121,6 +126,20 @@ func newDebugProcess(pid int, exepath string) (*Process, error) {
 		dbp.postExit()
 		return nil, ProcessExitedError{Pid: dbp.Pid, Status: exitCode}
 	}
+	for _, thread := range dbp.Threads {
```

Comment:
Need to add a comment here as to why we're suspending and then continuing. It's hard to tell from here that we're trying to drain all events.

It's probably worth putting this logic into a well named function so it's apparent to the reader what's going on.